docs/chapters/sdk/installing-the-sdk.rst

```diff
@@ -0,0 +1,29 @@
+Downloading and installing the SDK
+==================================
+
+Download the SDK for your hardware from the `PELUX website`_, it is quite big because it contains tho whole sysroot for the project in addition to some native tools, so depending on your internet connectivity it could take a couple of minutes.
+
+If you download it from a HTTP server then the file per default is not executable, we need to change it with `chmod`. After that is done we can run it, it's a self extracting shell script.
```

Comment:
I believe we have to discussed this but I had a dilemma about structuring the sentences  in the SWF: should we use the 2nd person singular or plural when addressing to the users or should the docs be written in a passive voice, e.g. : 

"When downloaded from the HTTP server, the SDK will not be executable and its permission has to be changed with `chmod`.